uint/src/uint.rs

```diff
@@ -321,15 +368,9 @@ macro_rules! impl_mulassign_from {
 
 #[inline(always)]
 #[doc(hidden)]
-pub fn mul_u32(a: (u64, u64), b: u64, carry: u64) -> (u64, u64) {
```

Comment:
This is a public function, so we probably need to support it (with a deprecation warning?).

Reply:
Technically, this is true, but it is only used for `construct_uint` macro, and `doc(hidden)` indicate it shouldn't be a part of the public interface.